### PR TITLE
Add XCPrivacy file to Geofence SDK

### PR DESCRIPTION
**Pull request by kushCT**
- Supports [CleverTap iOS SDK v6.1.0](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/6.1.0).
- Adds privacy manifests.

### CODE REVIEW

**Review by darshanclevertap — APPROVED** [review]

File: CleverTap-Geofence-SDK.podspec
Comment (nishant-clevertap) [inline CleverTap-Geofence-SDK.podspec:1]: Add xcprivacy in resources as done in core sdk.

File: Package.swift
Comment (nishant-clevertap) [inline Package.swift:1]: add copy PrivacyInfo.xcprivacy and test for SPM once again

Comment (kushCT) [reply Package.swift:1]: @nishant-clevertap I have done the changes as per comment. Please review it.

**Review by nishant-clevertap — APPROVED** [review]
LGTM

**Pull request merged**